projects/admin/src/app/record/detail-view/holding-detail-view/holding-detail-view.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record && record.metadata as data">
+  <ng-container [ngSwitch]="data.holdings_type">
```

Comment:
replace the switch by an if. It can be added to the parent.

Reply:
I used this strategy for future. If next PR's if we need to implement a detail view for another kind of serial, we just need to add a `ngSwitchCase` without any other changes.
At this time, I'm sure than only 'serial holding' could be access to this `ngSwitch` regarding the L55 from https://github.com/rero/rero-ils-ui/pull/272/files/b8262179bb708c6f17698961a9ce4e49ebf520bb#diff-fa97e2f4aacaa4e697c3f5f48bf43159#L55

Reply:
I used this method to prepare the future. In possible next PR's, If we need to implement a detailed view for another kind of holding we only need to add a `*ngSwitchCase` without any other changes.